app/controllers/robots_controller.rb

```diff
@@ -1,4 +1,6 @@
 class RobotsController < ApplicationController
+  skip_before_action :authenticate, only: :index
```

Comment:
I hadn't realised how simple this was to implement. I thought we'd have to control it on the ApplicationController. 

question - why do we need to specify to only apply it to `index`, when it's the only thing there?

Reply:
That's just a failsafe. The default should be for all pages to be authenticated, and so if we ever add any new actions to RobotsController we want to make sure they're authenticated unless we specifically add code to say that they aren't.